### DISCUSSION TITLE
코로나 백신 종류 선택 방법 문의드립니다!

### DISCUSSION THREAD

**Discussion by covid19bye** [changdoc/naver-vaccine-macro#143] [Q&A]
![백신](https://user-images.githubusercontent.com/88148373/127501775-5a9c782d-2637-4434-acfc-5156cc7742e8.jpg)
혹시 이렇게 모더나 눌러 놓으면 모더나 백신만 예약가능한가요??
아니면 모더나 클릭 후 다른 버튼을 눌러야 하는가요??

**Comment by changdoc** [comment]
네 저 상태면 잔여 백신이 발생했을때 모더나 잔여 수 발생했을 경우에만 해당 백신 코드로 예약 신청을 시도하도록 되어 있습니다.

**Reply by changdoc** [reply]
백신 종류 고르는 순간 저장을 수행하고 있어 다른 버튼 별도로 누를 필요 없습니다.

**Reply by covid19bye** [reply]
감사합니다!!
